packages/salesforcedx-vscode-core/src/commands/baseDeployRetrieve.ts

```diff
@@ -280,7 +280,7 @@ export abstract class RetrieveExecutor<T> extends DeployRetrieveExecutor<T> {
   protected async postOperation(
     result: RetrieveResult | undefined
   ): Promise<void> {
-    if (result) {
+    if (result && result.response.fileProperties !== undefined) {
```

Comment:
Can be done with
``` suggestion
    if (result?.response?.fileProperties !== undefined) {
```

Reply:
I put my condition with the other condition because that was where it originally was.  It's pretty old code so I just kept the status quo. 🙂

I would assume it's because putting the if statement earlier means Lines 284-288 wouldn't be run unnecessarily.